accounting/formatters/OsgScheddCpuRemovedFormatter.py

```diff
@@ -0,0 +1,82 @@
+
+
+    def get_table_title(self, table_file, report_period, start_ts, end_ts):
+        info = self.parse_table_filename(table_file)
+        # Format date(s)
+        start = datetime.fromtimestamp(start_ts)
+        if report_period in ["daily"]:
+            basename = Path(table_file).stem
+            start_date = start.strftime("%Y-%m-%d")
+            title_str = f"OSPool per {info['agg'].rstrip('s')} usage for jobs removed on <strong>{start_date}</strong>"
+        elif report_period in ["weekly", "monthly"]:
+            end = datetime.fromtimestamp(end_ts)
+            start_date = start.strftime("%Y-%m-%d")
+            end_date = end.strftime("%Y-%m-%d")
+            title_str = f"OSPool per {info['agg'].rstrip('s')} usage for jobs removed from <strong>{start_date} to {end_date}</strong>"
+        else:
+            end = datetime.fromtimestamp(end_ts)
+            start_date = start.strftime("%Y-%m-%d %H:%M:%S")
+            end_date = end.strftime("%Y-%m-%d %H:%M:%S")
+            title_str = f"OSPool per {info['agg'].rstrip('s')} usage for jobs removed from <strong>{start_date} to {end_date}</strong>"
+        return title_str
+
+    def get_subject(self, report_period, start_ts, end_ts, **kwargs):
+        # Format date(s)
+        start = datetime.fromtimestamp(start_ts)
+        if report_period in ["daily", "weekly", "monthly"]:
+            start_date = start.strftime("%Y-%m-%d")
+            subject_str = f"OSPool {report_period.capitalize()} Removed Jobs Usage Report {start_date}"
+        else:
+            end = datetime.fromtimestamp(end_ts)
+            start_date = start.strftime("%Y-%m-%d %H:%M:%S")
+            end_date = end.strftime("%Y-%m-%d %H:%M:%S")
+            subject_str = f"OSPool Removed Jobs Usage Report {start_date} to {end_date}"
+        return subject_str
+
+    def format_rows(self, header, rows, custom_fmts={}, default_text_fmt=None, default_numeric_fmt=None):
+        custom_fmts = {
+            "Min Hrs":    lambda x: f"<td>{hhmm(x)}</td>",
+            "25% Hrs":    lambda x: f"<td>{hhmm(x)}</td>",
+            "Med Hrs":    lambda x: f"<td>{hhmm(x)}</td>",
+            "75% Hrs":    lambda x: f"<td>{hhmm(x)}</td>",
+            "Max Hrs":    lambda x: f"<td>{hhmm(x)}</td>",
+            "Mean Hrs":   lambda x: f"<td>{hhmm(x)}</td>",
+            "Std Hrs":    lambda x: f"<td>{hhmm(x)}</td>",
+            "CPU Hours / Bad Exec Att": lambda x: f"<td>{float(x):.1f}</td>",
+            "Shadw Starts / Job Id":    lambda x: f"<td>{float(x):.2f}</td>",
+            "Exec Atts / Shadw Start":  lambda x: f"<td>{float(x):.3f}</td>",
+        }
+        rows = super().format_rows(header, rows, custom_fmts=custom_fmts, default_text_fmt=default_text_fmt, default_numeric_fmt=default_numeric_fmt)
+        return rows
+    
+    def get_legend(self):
+        custom_items = OrderedDict()
+        custom_items["CPU Hours / Bad Exec Att"] = "Average CPU Hours used in a non-final execution attempt"
```

Comment:
Following changes above
```suggestion
        custom_items["CPU Hours / Exec Att"] = "Average CPU Hours used in a non-final execution attempt"
```